---
xbee_address:
  base_station: 417E6274
  husky1: 41AF1B1A
  husky2: 41AF1A91
  jackal1: 417E628A
  jackal2: 41AF1C55
